src/fprime_gds/common/fpy/bytecode/SPEC.md

```diff
@@ -1,52 +1,62 @@
 # Directives
```

Comment:
This seems a duplicate to the C++ code.  We need to pick a single source of truth and link back to it.